build.sbt

```diff
@@ -22,115 +21,97 @@ val wiremockVersion = "3.0.0-beta-7"
+    "org.typelevel" %% "cats-core" % catsVersion,
+    "uk.gov.hmrc.mongo" %% s"hmrc-mongo-$playVersion" % hmrcMongoVersion,
+    "uk.gov.hmrc" %% s"play-frontend-hmrc-$playVersion" % playFrontendHMRCVersion,
+    "uk.gov.hmrc" %% s"crypto-json-$playVersion" % cryptoJsonVersion,
+    "org.jsoup" % "jsoup" % jsoupVersion
   )
-}
 
-lazy val microservice = Project(appName, file("."))
-  .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
-  .disablePlugins(JUnitXmlReportPlugin)
-  .settings(playSettings: _*)
-  .settings(scalaSettings: _*)
-  .settings(scalaVersion := currentScalaVersion)
-  .settings(scoverageSettings: _*)
-  .settings(defaultSettings(): _*)
-  .settings(majorVersion := 1)
-  .settings(scalacOptions += "-Wconf:cat=lint-multiarg-infix:silent")
-  .settings(scalacOptions += "-Xfatal-warnings")
-  .settings(
-    Test / Keys.fork := true,
-    Test / javaOptions += "-Dlogger.resource=logback-test.xml",
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
+val commonTestDependencies =
+  Seq(
+    "org.scalatestplus.play" %% "scalatestplus-play" % scalaTestPlusVersion,
+    "org.scalamock" %% "scalamock" % scalaMockVersion,
+    "org.pegdown" % "pegdown" % pegdownVersion,
+    "org.jsoup" % "jsoup" % jsoupVersion,
+    "org.mockito" % "mockito-core" % mockitoVersion,
+    "uk.gov.hmrc.mongo" %% s"hmrc-mongo-test-$playVersion" % hmrcMongoVersion
   )
-  .settings(
-    libraryDependencies ++= appDependencies,
-    retrieveManaged := true
+
+def testDependencies(scope: String = "test"): Seq[ModuleID] =
+  commonTestDependencies.map(_ % scope) ++ Seq(
+    "org.scalacheck" %% "scalacheck" % "1.18.0" % scope,
+    "org.scalatestplus" %% "scalacheck-1-15" % "3.2.11.0" % scope,
+    "uk.gov.hmrc" %% s"bootstrap-test-$playVersion" % bootstrapPlayVersion % scope,
+    caffeine,
+    "uk.gov.hmrc" %% s"crypto-json-$playVersion" % cryptoJsonVersion % scope
   )
-  .settings(
-    Test / Keys.fork := true,
-    scalaVersion := currentScalaVersion,
-    scalacOptions += "-Wconf:src=routes/.*:s",
-    Test / javaOptions += "-Dlogger.resource=logback-test.xml")
-  .settings(
-    Keys.fork := false,
-    TwirlKeys.templateImports ++= Seq(
-      "uk.gov.hmrc.govukfrontend.views.html.components.implicits._",
-      "uk.gov.hmrc.hmrcfrontend.views.html.helpers._",
-      "uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._"
-    ),
-    RoutesKeys.routesImport := Seq("enums.IncomeSourceJourney._", "models.admin._", "models.core._"),
-    scalacOptions += "-Wconf:cat=unused-imports:s,cat=unused-params:s"
+
+def itDependencies(scope: String = "test"): Seq[ModuleID] =
+  commonTestDependencies.map(_ % scope) ++ Seq(
+    "com.github.tomakehurst" % "wiremock" % wiremockVersion % scope,
+    caffeine
   )
-  .settings(resolvers ++= Seq(
-    Resolver.jcenterRepo
-  ))
+
+lazy val appDependencies: Seq[ModuleID] = compileDependencies ++ testDependencies()
```

Comment:
as above, if we can have a separate story for these changes this must be discussed/agreed on the TLs level what and why we are doing these;

Reply:
it's mostly a clean up of build.sbt lots of repeated option or something from what I remember, and we can likely move out the dependencies and coverage into it's own object to help keep the file smaller, cleaner and concise.

Reply:
absolutely fine, move these to tech debt story and reduce scope of the current changes to control change / simplify testing, as unlikely you would test all these in the current sprint; you can addresses these in the scope of the current/next sprint when needed as a part of a new story